src/components/network-map-tab.js

```diff
@@ -231,7 +231,7 @@ export const NetworkMapTab = ({
                 handleClose={closeChoiceVoltageLevelMenu}
                 onClickHandler={choiceVoltageLevel}
                 substation={choiceVoltageLevelsSubstation}
-                position={[position[0] + 200, position[1]]}
+                position={[position[0] + 100, position[1]]}
```

Comment:
Is this a magical number ? seems like it is :stuck_out_tongue:

Reply:
I changed it with a constant